src/styles/blocks.scss/header-nav.scss

```diff
@@ -0,0 +1,44 @@
+.nav {
+  &__list {
+    display: flex;
+    text-align: center;
+    list-style: none;
+    text-decoration: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__link {
+    display: flex;
+    color: #060b35;
+    text-decoration: none;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    text-transform: uppercase;
+    line-height: 60px;
+    height: 60px;
```

Comment:
The `height: 60px;` property seems to be unnecessary here. The height of the link should be determined by its content and padding.